ts_src/tests/fromBIP/valid.ts

```diff
@@ -3,18 +3,49 @@ import { Psbt } from '../../lib/psbt';
 import { fixtures } from '../fixtures/valid';
 import { transactionFromBuffer } from '../utils/txTools';
 
-for (const f of fixtures) {
-  tape(`Test: Should not throw`, t => {
-    let psbt: Psbt;
-    t.doesNotThrow(() => {
-      psbt = Psbt.fromBase64(f, transactionFromBuffer);
-    }, 'fromBase64');
-
-    t.doesNotThrow(() => {
-      const out = psbt.toBase64();
-      t.equal(out, f);
-    }, 'toBase64');
+interface ContainsBip32Paths {
+  bip32Derivation?: Array<{ path: string }>;
+  tapBip32Derivation?: Array<{ path: string }>;
+}
 
-    t.end();
-  });
+function getPaths(m: ContainsBip32Paths[]): string[][] {
+  return m.map(ele =>
+    [...(ele.bip32Derivation || []), ...(ele.tapBip32Derivation || [])].map(
+      ({ path }) => path,
+    ),
+  );
 }
+
+[true, false].forEach(bip32PathsAbsolute => {
+  for (const f of fixtures) {
+    tape(
+      `Test: Should not throw (bip32PathsAbsolute:${bip32PathsAbsolute})`,
+      t => {
+        let psbt: Psbt;
+        t.doesNotThrow(() => {
+          psbt = Psbt.fromBase64(f, transactionFromBuffer, {
+            bip32PathsAbsolute,
+          });
+          const paths = [
+            ...getPaths(psbt.inputs),
+            ...getPaths(psbt.outputs),
+          ].reduce((a, b) => [...a, ...b], []);
```

Comment:
I think it'd be slightly nicer to move this flatten into the `getPaths` function.